packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -656,6 +656,9 @@ function renderHook<
 // re-export everything
 export * from '@testing-library/react';
 
+// re-export everything from @testing-library/user-event
+export * from '@testing-library/user-event';
```

Comment:
I would suggest to namespace the exports as `userEvents`, like for hooks to avoid possible conflicts.